src/bin/bench-tps.rs

```diff
@@ -575,6 +572,9 @@ fn main() {
         socketaddr!("127.0.0.1:8001")
     };
 
+    let mut drone_addr = network.clone();
```

Comment:
Ah, let's take this one step further and add a `--drone` argument for an IP:port.  In `./multinode-demo/client.sh` we can then provide `--drone` using the same IP provided to `--network` for now